src/components/projects/project-edit-form/project-edit-container/project-edit-wp-list/project-edit-wp-list.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Card } from 'react-bootstrap';
+import { WorkPackage } from 'utils';
+import { wbsPipe, weeksPipe } from '../../../../../shared/pipes';
+import PageBlock from '../../../../shared/page-block/page-block';
+import styles from './project-edit-wp-list.module.css';
+
+interface ProjectEditWorkPackagesListProps {
+  workPackages: WorkPackage[];
+}
+
+const ProjectEditWorkPackagesList: React.FC<ProjectEditWorkPackagesListProps> = ({
+  workPackages
+}) => {
```

Comment:
can we just not have this? the old work package list component should work perfectly fine, no?

Reply:
the old one includes the collapsible summary plus active links to those work packages, is that what we want?

Reply:
i only remade it like this to get just the header info of the work packages and not have the dropdown stuff, but if we're fine with having the dropdown info i can scrap this and use that old list

Reply:
yeah I'm fine with the dropdown and the dropdown would honestly probably be a better user experience